### PR TITLE
Test latest PG 11.10 and 12.5

### PR DESCRIPTION
**Pull request by k-rus**

### CODE REVIEW

**Review by pmwkaa — APPROVED** [review]

File: .github/workflows/alpine-32bit-build-and-test.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ jobs:
...
       matrix:
-        pg: [ 11.9, 12.4 ]
+        pg: [ 11.10, 12.5 ]
         build_type: [ Debug ]
         include:
-          - pg: 11.9
-            ignores: append-11 chunk_adaptive-11 continuous_aggs_bgw_drop_chunks remote_txn transparent_decompression-11
-          - pg: 12.4
+          - pg: 11.10
+            ignores: append-11 chunk_adaptive-11 continuous_aggs_bgw_drop_chunks remote_txn transparent_decompression-11 continuous_aggs_insert continuous_aggs_multi continuous_aggs_concurrent_refresh
```
Comment (gayyappan) [inline .github/workflows/alpine-32bit-build-and-test.yaml:25]: Do we know why these additional continuous-aggs tests are failing? Shouldn't this be fixed?

Comment (k-rus) [reply .github/workflows/alpine-32bit-build-and-test.yaml:25]: @gayyappan The added tests `continuous_aggs_insert`, `continuous_aggs_multi`, `continuous_aggs_concurrent_refresh` are failing due to different log output in the isolation test (no more session prefix). It looks to me as regression in PostgreSQL in the case of 32bit build of PG 11.10. The diff for the tests:
```diff
--- /__w/timescaledb/timescaledb/tsl/test/isolation/expected/continuous_aggs_concurrent_refresh.out	2020-11-16 12:16:42.468949596 +0000
1349
+++ /__w/timescaledb/timescaledb/build/tsl/test/isolation/results/continuous_aggs_concurrent_refresh.out	2020-11-16 12:18:17.434368377 +0000
1350
@@ -282,7 +282,6 @@
1351
     ROLLBACK;
1352
 
1353
 step R1_refresh: <... completed>
1354
-R2: NOTICE:  continuous aggregate "cond_10" is already up-to-date
1355
 step R2_refresh: <... completed>
1356
 step S1_select: 
1357
     SELECT bucket, avg_temp
1358

1359
======================================================================
1360

1361
--- /__w/timescaledb/timescaledb/tsl/test/isolation/expected/continuous_aggs_insert.out	2020-11-16 12:16:42.468949596 +0000
1362
+++ /__w/timescaledb/timescaledb/build/tsl/test/isolation/results/continuous_aggs_insert.out	2020-11-16 12:18:18.446381910 +0000
1363
@@ -6,7 +6,7 @@
1364
 step Refresh: CALL refresh_continuous_aggregate('continuous_view', NULL, 15); <waiting ...>
1365
 step UnlockInvalThrEx: ROLLBACK;
1366
 step Refresh2: <... completed>
1367
-R: NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1368
+NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1369
 step Refresh: <... completed>
1370
 
1371
 starting permutation: Ib LockCagg I1 Refresh Ic UnlockCagg
1372
@@ -136,7 +136,7 @@
1373
 step LockInvalThrEx: BEGIN; LOCK TABLE _timescaledb_catalog.continuous_aggs_invalidation_threshold ; <waiting ...>
1374
 step Ic: COMMIT; <waiting ...>
1375
 step UnlockMatInval: ROLLBACK;
1376
-R1: NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1377
+NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1378
 step Refresh1: <... completed>
1379
 step LockInvalThrEx: <... completed>
1380
 step UnlockInvalThrEx: ROLLBACK;
1381
@@ -160,7 +160,7 @@
1382
 31             
1383
 step Sc: COMMIT;
1384
 step UnlockInval: ROLLBACK;
1385
-R: NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1386
+NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1387
 step Refresh: <... completed>
1388
 
1389
 starting permutation: I1 Refresh LockInval Sb S1 Refresh Sc UnlockInval
1390
@@ -175,5 +175,5 @@
1391
 step Refresh: CALL refresh_continuous_aggregate('continuous_view', NULL, 15); <waiting ...>
1392
 step Sc: COMMIT;
1393
 step UnlockInval: ROLLBACK;
1394
-R: NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1395
+NOTICE:  continuous aggregate "continuous_view" is already up-to-date
1396
 step Refresh: <... completed>
1397

1398
======================================================================
1399

1400
--- /__w/timescaledb/timescaledb/tsl/test/isolation/expected/continuous_aggs_multi.out	2020-11-16 12:16:42.468949596 +0000
1401
+++ /__w/timescaledb/timescaledb/build/tsl/test/isolation/results/continuous_aggs_multi.out	2020-11-16 12:18:18.914388161 +0000
1402
@@ -98,7 +98,7 @@
1403
 step Refresh1: CALL refresh_continuous_aggregate('continuous_view_1', NULL, 30); <waiting ...>
1404
 step Refresh2: CALL refresh_continuous_aggregate('continuous_view_2', NULL, NULL);
1405
 step UnlockMat1: ROLLBACK;
1406
-R1: NOTICE:  continuous aggregate "continuous_view_1" is already up-to-date
1407
+NOTICE:  continuous aggregate "continuous_view_1" is already up-to-date
1408
 step Refresh1: <... completed>
1409
 step Refresh1_sel: select * from continuous_view_1 where bkt = 0 or bkt > 30
1410
 bkt            cnt            
```

**Review by mkindahl — COMMENTED** [review]
If there is a test failure and the change it benign, shouldn't the result files be updated to match?

**Comment by k-rus** [comment]
> If there is a test failure and the change it benign, shouldn't the result files be updated to match?

@mkindahl 
The difference happens only on 32bit and PG 11.10. It works as expected on 64 bit. Can you clarify how you imagine to provide different result only for this case?

**Comment by mkindahl** [comment]
> > If there is a test failure and the change it benign, shouldn't the result files be updated to match?
> 
> @mkindahl
> The difference happens only on 32bit and PG 11.10. It works as expected on 64 bit. Can you clarify how you imagine to provide different result only for this case?

No, I cannot.

I find it very strange that it provides different results on the two platforms, but if we have a situation where we should ignore failing tests on some platforms but not on others, it will be very hard to work with the repository and we are bound to introduce problems that the tests are supposed to prevent.

**Review by mkindahl — APPROVED** [review]

**Pull request merged**